lib/tasks/print_statuses.rake

```diff
@@ -0,0 +1,30 @@
+desc "Print statuses of Tasks, Issues & Risks"
+task :print_statuses => :environment do
+  puts "*"*89, "TASKS", "*"*89
+  Task.all.each do |task|
```

Comment:
(Optional)
Use in_batches(of: 1000) to speed the iteration. 
Task.in_batches(of: 1000) do |tasks|
   tasks.find_each do |task|
      ....
   end
end

Reply:
Yes, I thought of it as well.
Let me do it now.